static/sass/_v1_pattern_navigation.scss

```diff
@@ -0,0 +1,181 @@
+.p-navigation {
```

Comment:
I would like to maintain the same level of modularisation all the way through the stack. From Vanilla to theme to site. That way we have a standard way structuring Scss.